python-sdk/src/astro/files/locations/base.py

```diff
@@ -132,6 +132,14 @@ def exists(self) -> bool:
         except OSError:
             return False
 
+    def databricks_settings(self) -> dict:
```

Comment:
It feels odd to have a method specific to `databricks` on the base class. Do you see any alternative ways of handling this?

Reply:
@tatiana I really don't. We need to have databricks settings specific to each file type. I don't see this as weird because databricks requires settings that others don't.

Reply:
I spoke with @tatiana and this conversation can be resolved as soon as I create a ticket to look into how best to support `databricks_settings` and link it here

Reply:
https://github.com/astronomer/astro-sdk/issues/1429 Added issue here